app/models/user.rb

```diff
@@ -485,10 +485,10 @@ def clear_account!
       self[field] = nil
     end
     [:getting_started,
-     :disable_mail,
      :show_community_spotlight_in_stream].each do |field|
```

Comment:
I can't seem to spot an existing test for these fields, that's what I mean with backfilling ;)